WooCommerce/Classes/ViewRelated/Orders/Order Filters/Order Status Filter/OrderStatusFilterViewController.swift

```diff
@@ -18,11 +18,15 @@ final class OrderStatusFilterViewController: UIViewController {
 
     private var selected: [OrderStatusEnum]
 
+    private var allowedStatuses: [OrderStatus] = []
```

Comment:
nit: can this be `let`?
```suggestion
    private let allowedStatuses: [OrderStatus]
```